apps/desktop/src/components/ui/warcraftcn/styles/warcraft.css

```
@@ -0,0 +1,982 @@
+@import url("https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap");
+
+.fantasy {
+  font-family: "Cinzel", serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Cinzel" (font-family-name-quotes)

(font-family-name-quotes)